src/swap/saga.test.ts

```diff
@@ -104,29 +89,14 @@ describe(swapSubmitSaga, () => {
       .provide([
         [select(walletAddressSelector), mockAccount],
         [call(getContractKit), contractKit],
-        [call(getConnectedUnlockedAccount), mockAccount],
-        [call(fetchWithTimeout, executeSwapUri), throwError(new Error('Error Fetching'))],
-      ])
-      .put(swapApprove())
-      .put(swapError())
-      .run()
-  })
-
-  it('should set swap state correctly when response ok is false', async () => {
-    await expectSaga(swapSubmitSaga, mockSwap)
-      .withState(store.getState())
-      .provide([
-        [select(walletAddressSelector), mockAccount],
-        [call(getContractKit), contractKit],
-        [call(getConnectedUnlockedAccount), mockAccount],
-        [call(fetchWithTimeout, executeSwapUri), { ok: false }],
+        [call(getConnectedUnlockedAccount), throwError(new Error('fake error'))],
```

Comment:
nit: we're testing for a unlock account error, i wonder if it's more realistic to test for an error on `sendTransaction`?

Reply:
Fixed in 5abe29b41bd423565ee1fd3a31106bd8842e1964!